.travis.yml

```diff
@@ -41,7 +42,7 @@ before_script:
   # Decodes to e.g. `export VARIABLE=VALUE`
   - $(base64 --decode <<< "ZXhwb3J0IFNBVUNFX0FDQ0VTU19LRVk9YTAzNTM0M2YtZTkyMi00MGIzLWFhM2MtMDZiM2VhNjM1YzQ4")
   - $(base64 --decode <<< "ZXhwb3J0IFNBVUNFX1VTRVJOQU1FPXJ1YnlvbnJhaWxz")
-  - redis-server --bind 127.0.0.1 --port 6379 --requirepass 'password' --daemonize yes
+  - redis-server --bind 127.0.0.1 --port 6380 --requirepass 'password' --daemonize yes
```

Comment:
Separate Redis for Action Cable tests which specifically want to work against a non-default config.

Use the default `redis-server` service for Active Support cache store tests.

Reply:
Maybe we should have the test suite spin this up & down directly? Having to set a password on my local redis to get the tests to pass was annoying, but making everyone manually run a second instance seems even worse.

Reply:
Yeah. I'd rather change the Action Cable tests separately. The point of this was to demonstrate that AC would accept user credentials for Redis. A mocked test should be sufficient. Then we can use a default Redis config for all tests.

Reply:
Done in #31136